ladder/teams/README.md

```diff
@@ -0,0 +1,51 @@
+will propogate updates from the team files into the Cilium Organization's team membership using the GitHub API.
+
+## Are You a Team Member or a Mentor? Choose, but Choose Wisely.
+With membership comes both priviledges and responsibilities.  One of the responsibilities of being on an active Cilium team member is 
+being available for automatic pull request review assignment for files owned by that team. 
+
+Over time as contributors transition from one team to another inside the project, while not active on a team any longer, 
+their historic experience may be needed to help review proposed changes with functionality they have insight into.  In these cases these 
+individuals can be added to the team's mentor list. Mentors are no longer part of the auto-assigned PR review rotation, but they can still keep team 
+approval access. Mentors of a team are expected to provide guidance to others in the team and seek reviews they should participate in for the code owner area.
+
+## Don't Forget to Set Your GitHub Status
+When active team members are not available for an extended period of time (for example they are on vacation for several days), 
+they are encouraged to set themselves as busy in their [GitHub status](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#setting-a-status). 
+
+Note: Not everyone should be busy all at the same time on a team if it can be avoided. Cilium Team Management automation puts 
+some guardrails on this and lets the organization know if there aren't sufficient number of reviewers. Admins (or someone 
+delegated to keep tabs on this) will periodically check to see if teams are meeting the expectations of number of people 
+available for review assignment and will reach out to teams if there appears to be a chronic shortfall.
+
+## Team File format
+
+Each team that wants to self-service their team members (and mentors!) can have a file in this directory named `TEAM_NAME.yaml`
+The file contents as the form of
+```
+members:
+ - username_01
+ - username_02
+ - username_03
+
+mentors:
+ - username_02
+ - username_04
+```
+
+In this mockup team file, `username_02` is both a member and a mentor and would be excluded from auto-assignment of PR reviews, but would 
+have other team rights such as commit and approval for changes to files owned by this team.
+
+`username_01` and `username_03` are active team members particiating in the automatic PR review rotation.
+
+`username_04` is listed as a mentor, but not a member. This is here as recognition that historically the person participated in the team 
+and is willing to be contacted for historical experience when needed, but doesn't have normal team member access to approve.
+Listing non-active members as mentors is a way to keep institutional knowledge as to who provide insight about code functionality to newer 
```

Comment:
```suggestion
Listing non-active members as mentors is a way to keep institutional knowledge as to who historically provided insight about code functionality to newer 
```

Reply:
I think it would make more sense to go to a mentor than a non-active member and that's what I'm trying to convey here.

Reply:
I think "non-active" is also doing a bit of work here, maybe making this less clear than it could be. The way I think of it is when someone's primary focus is no longer in this area, not that they won't respond to questions for instance.